angular.json

```diff
@@ -67,7 +67,8 @@
         "serve": {
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
-            "browserTarget": "rightfoot-payments-demo:build"
+            "browserTarget": "rightfoot-payments-demo:build",
+            "proxyConfig": "src/proxy.conf.json"
```

Comment:
While this way works, I think it's better to modify this behavior in the package.json file, like we do in browserclient, for example.
Under the `"scripts"` you can replace `start` script with this, it should also work, without modifying the behavior of ng serve. Then you just run `yarn start`.
`"start": "ng serve --proxy-config proxy.conf.json",`
Or was there a specific reason you choose to set `proxyConfig` here?